src/backend/internal/models/user.go

```diff
@@ -13,6 +13,7 @@ type User struct {
 	LastName  string `gorm:"column:lastname;type:varchar(255);not null" json:"lastname" binding:""`
 	Email     string `gorm:"column:email;unique;type:varchar(255);not null" json:"email" binding:"required,email"`
 	Password  string `gorm:"column:password;type:varchar(255);not null" json:"password" binding:"required"`
+	IsAdmin   bool `gorm:"column:isAdmin;type:BOOLEAN" json:"isAdmin"`
```

Comment:
I think a better approach to this will be to use the Role table to indicate Admin or Employee.

Reply:
I updated the PR to use https://github.com/casbin/casbin